soc/riscv/starfive_jh71xx/jh71xx/linker.ld

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2024 Pratik Farkase <pratik.farkase@wsisweden.com>
+ * Copyright (c) 2024 Sigma Connectivity WSI AB
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arch/riscv/common/linker.ld>
```

Comment:
Instead of adding this file, you should use the following line in a CMake script:
```set(SOC_LINKER_SCRIPT ${ZEPHYR_BASE}/include/zephyr/arch/riscv/common/linker.ld CACHE INTERNAL "")```

Reply:
Removed the redundant `linker.ld` as `CMakeLists.txt` script already contains the `set(SOC_LINKER_SCRIPT ${ZEPHYR_BASE}/include/zephyr/arch/riscv/common/linker.ld CACHE INTERNAL "")`.